code/modules/food_and_drinks/food/condiment.dm

```diff
@@ -101,6 +101,12 @@
 	list_reagents = list(/datum/reagent/consumable/enzyme = 50)
 	fill_icon_thresholds = null
 
+/obj/item/reagent_containers/food/condiment/enzyme/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'>40 Milk, 5 Enzyme and you got cheese.</span>"
```

Comment:
Calculate this from the recipe instead of writing it into the text.

Reply:
To be clear, I mean the numbers. If you can automate the names too, that's good, but the numbers must be automatic.

Reply:
Here and everywhere else.

If you want, you can even make a variable for it that formats it, like:

```dm
exact_examine_description = "{milk} milk, {universal_enzyme} enzyme, and you got cheese."
```

Reply:
@Mothblocks comments need addressing